src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -354,9 +356,9 @@ def _serialise_resource_element(self) -> etree._Element:
         return etree.Element(f"{DASCH_SCHEMA}region", attrib=attribs, nsmap=XML_NAMESPACE_MAP)
 
     def _serialise_values(self) -> list[etree._Element]:
-        return [
-            LinkValue(value=self.region_of, prop_name="isRegionOf", resource_id=self.res_id).serialise(),
-        ]
+        return serialise_values(
```

Comment:
This is a little odd but I am planning on improving that in the future, but for now it works and allowed me to remove the `.serialise()` function completely.